pkg/loop/internal/core/services/capability/capabilities_registry.go

```diff
@@ -21,6 +22,44 @@ type capabilitiesRegistryClient struct {
 	grpc pb.CapabilitiesRegistryClient
 }
 
+func toPeerID(peerID []byte) p2ptypes.PeerID {
+	return p2ptypes.PeerID(peerID)
+}
+
+func toDON(wf *pb.DON) capabilities.DON {
```

Comment:
/nit `wf` seems misleading. is there anything workflow specific about the input arg?

Reply:
That's an artifact when i was working with just the workflow don, fixed